tools/plink_load_variant_file.cwl

```diff
@@ -25,23 +29,33 @@ inputs:
   const_fid: { type: 'string?', inputBinding: { position: 2, prefix: "--const-fid" }, doc: "converts sample IDs to within-family IDs while setting all family IDs to a single value" }
   id_delim: { type: 'string?', inputBinding: { position: 2, prefix: "--id-delim" }, doc: "causes sample IDs to be parsed as <FID><delimiter><IID>" }
   vcf_idspace_to: { type: 'string?', inputBinding: { position: 2, prefix: "--vcf-idspace-to" }, doc: "convert all spaces in sample IDs to this character" }
-  biallelic_only: { type: ['null', { type: enum, name: "biallelic_only", symbols: ["base", "strict", "list"] }], inputBinding: { position: 2, prefix: "--biallelic-only", shellQuote: false, valueFrom: "$(self == 'base' ? '' : self)"}, doc: "Use to skip all variants where at least two alternate alleles are present in the dataset. The 'strict' mode will indiscriminately skip variants with 2+ alternate alleles listed even when only one alternate allele actually shows up. The 'list' mode will dump a list of skipped variant IDs to plink.skip.3allele." }
+  biallelic_only:
```

Comment:
Interesting way to do this...considering implementing for STAR as it has a similar arg structure for some of them

Reply:
Yeah this is bugged a bit on Cavatica so I had to do the workaround with arguments. Hopefully they fix it and it'll be even better.